File: leetcode/two_sum.py
class Solution:
    def twoSum(self, nums, target):
        for i in range(0, len(nums)-1):
            for j in range(i+1, len(nums)):
                list = []
                if nums[i]+nums[j] == target:
                    list.append(i)
                    list.append(j)
                    return list

File: leetcode/two_sum.rb
# frozen_string_literal: true

# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer[]}
def two_sum(nums, target)
  hash = {}
  nums.each_with_index do |n, i|
    return hash[target - n], i if hash[target - n]

    hash[n] = i
  end
end
